packages/pwa-kit-create-app/assets/bootstrap/js/config/default.js.hbs

```diff
@@ -51,9 +70,10 @@ module.exports = {
         // The default site for your app. This value will be used when a siteRef could not be determined from the url
         defaultSite: '{{answers.project.commerce.siteId}}',
         // Provide aliases for your sites. These will be used in place of your site id when generating paths throughout the application.
-        // siteAliases: {
-        //     RefArch: 'us'
-        // },
+        siteAliases: {
+            RefArch: 'us',
+            RefArchGlobal: 'global'
+        },
```

Comment:
🤔 Is this change meant only for demo site? Or we wanted it to be new defaults for everyone?

Reply:
This is meant to be for the demo site but I don't see the harm in having these enabled by default since they specifically target RefArch / RefArchGlobal. Most projects will go on to replace these with their own site ids.

Reply:
I updated the template to only apply this for the demo site